### ISSUE TITLE
pad_height and pad_height calculations

### ISSUE THREAD

**Issue by prajwal967** [keunwoochoi/residual_block_keras#10]
The code that calculates pad_height and pad_width in the design_for_residual_blocks() function return a NoneType. How can this be corrected

**Comment by keunwoochoi** [comment]
Yep.. the function is bit arbitrarily designed. Could you let me know the input arguments you used?

**Comment by prajwal967** [comment]
Used the MNIST dataset, (28,28). Didnt change anything else

**Comment by xuetaohao** [comment]
Hi, when I use MINIST as input, it shows "ValueError: Only layers of same output shape can be merged using sum mode. Layer shapes: [(None, 128, 32, 32), (None, 128, 32, 128)]" on y = merge([shortcut_y, conv_y], mode='sum')
